### PR TITLE
Rename 'Shared Signals and Events Framework' to 'Shared Signals Framework' and 'SSE' to 'SSF'

### PR DESCRIPTION
**Pull request by FragLegs**
Once this is merged, I will update the website to reflect the docs path changes

### CODE REVIEW

File: docs/how-to-create-a-stream.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # How to Create a Stream
 
-The Shared Signals and Events spec leaves it up to the implementer how to
+The Shared Signals spec leaves it up to the implementer how to
```
Comment (deshmukhrajvardhan) [inline docs/how-to-create-a-stream.md:3]: should this be Shared Signals Framework spec?

File: docs/decoding-sets.md [hunk 2]
```diff
@@ -5,10 +5,11 @@ SETs can be encoded with various different algorithms, as [described here](https
 ## Retrieving the JWKS
 To decode SETs from a transmitter, we need the transmitter's JWKS [(JSON Web Key Set)](https://datatracker.ietf.org/doc/html/rfc7517#section-5):
 ```py
-sse_config_response = requests.get(
-    'https://transmitter.most-secure.com/.well-known/sse-configuration')
-sse_config = sse_config_response.json()
-jwks_uri = sse_config['jwks_uri']  # for example, https://transmitter.most-secure.com/jwks.json
+ssf_config_response = requests.get(
+    'https://transmitter.most-secure.com/.well-known/sse-configuration'
```
[the remark this answers is not in the document]
Comment (FragLegs) [reply docs/decoding-sets.md:9]: Yes, I would expect so.

File: docs/how-to-create-a-stream.md (hunk 1 above)
Comment (FragLegs) [reply docs/how-to-create-a-stream.md:3]: Meh. I think calling it "Shared Signals" sometimes is just fine

File: examples/README.md [hunk 3]
```diff
@@ -17,23 +17,23 @@ of the OpenAPI spec described in [transmitter_spec](../transmitter_spec/openapi.
...
 Verification Event from the transmitter. This allows you to force the transmitter
 to generate an event to demonstrate the PUSH capability.
 
-## Trigger and test SSE events other than verification
+## Trigger and test SSF events other than verification
 After you have the Swagger UI up, as per instructions [here](transmitter/README.md).
 You can use the out-of-band (**NOT** openid standard) transmitter endpoint `trigger-event`,
-to trigger any type of openid-sse event 
-([CAEP](https://openid.net/specs/openid-caep-specification-1_0-ID1.html) or [RISC](https://openid.net/specs/openid-risc-profile-specification-1_0-01.html)) 
+to trigger any type of shared signals event
```
[the remark this answers is not in the document]
Comment (FragLegs) [reply examples/README.md:35]: Same comment as above

**Pull request merged**